mkdocs.yml

```diff
@@ -8,8 +8,11 @@ theme:
 
 extra_css:
   - 'stylesheets/custom-style.css'
+  - 'stylesheets/downloads.css'
 extra_javascript:
   - 'javascripts/extra.js'
+  - 'https://unpkg.com/enigma.js/enigma.min.js'
```

Comment:
Define a specific version of enigma.js instead of the latest one, since that may cause issues in the future.